_events/sr2019/competition.md

```diff
@@ -18,22 +18,20 @@ On Sunday morning, the league matches will continue. Once these have concluded,
 
 ## Tickets and Media Consent
 
-Competitors need to bring along a ticket to be allowed entry. Anyone under the
-age of 18 must be accompanied by an adult. Student Robotics will announce when
-tickets are available.
+Competitors need to bring along a ticket to be allowed entry. Anyone under the age of 18 must be accompanied by an adult. To obtain a ticket, competitors need to sign a [media consent form][mcf-link] and [send back to us][teams-contact]. Those under the age of 16 will need to get a parent or guardian to sign on their behalf. Once processed, you'll receive an automated email with a link to your ticket.
 
-All competitors and visitors will need to provide media consent. More details to come soon.
+Media consent forms will also be available at reception.
 
 ## Location
 
 The competition will be held in the Students' Union building at the [University of Southampton's Highfield campus][soton-campus-directions]:
 
-Student's Union  
-University of Southampton  
-University Road  
-Southampton  
-SO17 1BJ  
-United Kingdom  
+Student's Union
+University of Southampton
+University Road
+Southampton
+SO17 1BJ
+United Kingdom
```

Comment:
The whitespace change in this address will break the rendering, unfortunately. The spaces indicate to Jekyll that the lines should be line-break separated.